src/woff2/collection.rs

```diff
@@ -4,13 +4,13 @@ use crate::woff2::{PackedU16, TableDirectoryEntry, Woff2Font};
 
 #[derive(Debug)]
 pub struct Directory {
-    version: u32,
+    pub version: u32,
```

Comment:
I don't have a lot of context on the dominant style in Allsorts so feel free to ignore me or disagree, but I wonder if `#[allow(unused)]` or whatever the warning was might be more appropriate here? Alternatively we could expose a method? This would avoid exposing these to the public API prematurely, and also mixing `pub` and non-`pub` fields in our API (see [this post](https://matklad.github.io//2022/05/29/binary-privacy.html) for more information on this rule of thumb).

Reply:
(also feel free to ignore because the dependency bump is more important than this minor quibble)

Reply:
I agree, I don't think we want to make them public just to silence the warnings.

Reply:
> but I wonder if #[allow(unused)] or whatever the warning was might be more appropriate here

Yep, that sounds better. Will do.